packages/cli/src/actions/buyCompleteSet.ts

```diff
@@ -11,13 +12,17 @@ const buyCompleteSet = async (opts: Options): Promise<void> => {
   const { endpoint, marketId, amount, seed } = opts;
 
   const sdk = await SDK.initialize(endpoint);
-
+  
   const signer = util.signerFromSeed(seed);
+  console.log(`Buy complete set of ${amount} market ${marketId} outcomes for ${signer.address} on network ${endpoint}`);  
 
   const market = await sdk.models.fetchMarketData(Number(marketId));
   const res = await market.buyCompleteSet(signer, Number(amount));
 
-  console.log(res);
+  console.log('res:', res);
```

Comment:
I guess this is debug output now and can be removed. At some point it might be good to use a more sophisticated logger that would allow us to set different logging levels like debug, info, error, etc.

```suggestion
```